spec/src/main/asciidoc/chapters/applications.asciidoc

```diff
@@ -56,4 +56,34 @@ i.e., if a subclass or implementation method has any MVC or JAX-RS annotations t
 
 [tck-testable tck-id-class-vs-iface]#Annotations on a super-class take precedence over those on an implemented interface#. 
 The precedence over conflicting annotations defined in multiple implemented interfaces is implementation dependent. 
-Note that, in accordance to the JAX-RS rules, inheritance of class or interface annotations is not supported. 
+Note that, in accordance to the JAX-RS rules, inheritance of class or interface annotations is not supported.
+
+[[configuration_in_mvc]]
+Configuration in MVC
+~~~~~~~~~~~~~~~~~~~~
+
+Implementations MUST support configuration via the native JAX-RS configuration mechanism but MAY support other configuration sources.
+
+There are concrete configurations, that all MVC the implementations are `REQUIRED` the support such as : 
+
+- `CsrfOptions.IMPLICIT`
```

Comment:
I think this list is confusing because it mixes constants for configuration property names (like `ViewEngine.VIEW_FOLDER`) with property values (like the enum `Csrf.CsrfOptions`) and even other constants (like `ViewEngine.DEFAULT_VIEW_FOLDER`)

What do you think about just listing the allowed configuration property names here. AFAIK these are:

 * `ViewEngine.VIEW_FOLDER`
 * `Csrf.CSRF_PROTECTION`

But maybe there are others? Perhaps you could double check the spec and the API classes?

Reply:
Hi @chkal ,

I see this now . thanks for review .

Reply:
HI, 

I finding in class Priorities some values how :  int DEFAULT = 1000; 

these values wes part of configuration  required ?

Reply:
No, `Priorities` is not a configuration key. So we should not include it here.